### DISCUSSION TITLE
Missing skia winit vulkan backend?

### DISCUSSION THREAD

**Discussion by qarmin** [slint-ui/slint#3799] [General]
When looking at https://slint.dev/releases/1.2.2/docs/slint/src/advanced/backend_winit

I see that skia not supports vulkan, but  in kms backed it supports - https://slint.dev/releases/1.2.2/docs/slint/src/advanced/backend_linuxkms

Renderer name | Supported/Required Graphics APIs | SLINT_BACKEND value to select renderer
-- | -- | --
FemtoVG | OpenGL | winit-femtovg
Skia | OpenGL, Metal, Direct3D | winit-skia
software | Software-rendering, no GPU required | winit-software

Renderer name | Required Graphics APIs | SLINT_BACKEND value to select renderer
-- | -- | --
FemtoVG | OpenGL ES 2.0 | linuxkms-femtovg
Skia | OpenGL ES 2.0, Vulkan | linuxkms-skia-opengl or linuxkms-skia-vulkan

so it is(or will be) possible to use skia with vulkan with winit?

**Comment by tronical** [comment]
Oh that’s an oversight in the docs. If you enable the `renderer-skia-vulkan` feature and select the winit backend with Skia (`SLINT_BACKEND=winit-skia`), then Slint will attempt to render with Skia and Vulkan.

I’ve tested this on x11 and macOS. I could not get my Vulkan drivers to work with VMware fusion on Windows. With Wayland I could also get it working but there was a strange issue with the swap chain images not being cleared with the background color. I tried for about 30 minutes to find out what’s causing it but could not.
